File: SiHyeon/week11/푸드 파이트 대회.cpp
#include <string>
#include <vector>

using namespace std;

string solution(vector<int> food)
{
    string answer = "0";
    for (int i = food.size() - 1; i > 0; i--)
    {
        string temp = "";
        temp.append((int)food[i] / 2, i + '0');
        answer = temp + answer + temp;
    }
    return answer;
}

File: yejee/week11/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        StringBuilder answer = new StringBuilder();
        
        for(int i = 1 ; i < food.length ; i++){
            answer.append(Integer.toString(i).repeat(food[i]/2));
        }
        
        return answer.toString() + "0" + answer.reverse().toString();
    }
}